src/reconcile/rec_visibility.c

```diff
@@ -40,6 +40,8 @@ __rec_update_save(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WT_INSERT *ins, WT
     supd->tw = *tw;
     supd->restore = supd_restore;
     ++r->supd_next;
+    if (onpage_upd != NULL || supd_restore)
```

Comment:
Can you add a comment here on why we are ignoring the tombstones?

Reply:
This is just to match what we do in __rec_calc_upd_memsize

Reply:
done